src/components/Navbar/Navbar.tsx

```diff
@@ -31,45 +31,49 @@ const Navbar = () => {
   return (
     <div className="nav-bar-main">
       <div>
-        <NavLink to="/">
-          <div className="nav-bar-logo">
+        <NavLink to="/" className="nav-link">
+          <div className="nav-bar--logo">
             <img src={Logo} alt="Xtreme cars logo" />
             <h5>XTREME CARS</h5>
           </div>
         </NavLink>
       </div>
 
-      <div className="nav-bar-car-btn-group">
+      <div className="nav-bar--car-btn-group">
         <div className="nav-bar-car-link">
-          <NavLink to="/">All Cars</NavLink>
+          <NavLink to="/" className="nav-link">
+            All Cars
+          </NavLink>
         </div>
 
         <div className="nav-bar-car-link">
-          <NavLink to="/compare-cars">Compare Cars</NavLink>
+          <NavLink to="/compare-cars" className="nav-link">
+            Compare Cars
+          </NavLink>
         </div>
       </div>
 
       <div className="nav-bar-right">
-        <div className="cart-div">
-          <NavLink to="/Wishlist">
-            <img
-              className="nav-bar-cart-img"
-              src={wishListIcon}
-              alt="wishList"
-            />
-            {wishCarsCount}
+        <div className="nav-cart-container">
+          <NavLink to="/Wishlist" title="Wishlist" className="nav-link">
+            <img className="nav-bar-wish-icon" src={wishlist} alt="wishlist" />
```

Comment:
this for e.g is your element 
hence an appropriate name would be `nav-cart-contianer__icon`

Reply:
Now lets say if we want to for some reason programattically switch between having this wichlist image with and without borders, we would  create 2 seperate classes for the same
`nav-cart-contianer__icon--borderless`
`nav-cart-contianer__icon--bordered`